.semversioner/next-release/patch-20241230224307150194.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "patch",
```

Comment:
Let's make this a `minor` release. Given the prompt change, we'll want people to rerun `graphrag init` to get that update on disk